### ISSUE TITLE
Bug: Clickable CARDS are not passing accessibility tests

### ISSUE THREAD

**Issue by jeremyzilar** [GSA/digitalgov.gov#1938]
At the moment, we have a lot of "cards" on the site that package information and make it clickable to that page or source. For example, we are using cards in the news feed https://digital.gov/news/ and on the events page https://digital.gov/events/ .

One of the problems with the way they are implemented is that they're not passing the accessibility checker tests...

![image](https://user-images.githubusercontent.com/395641/73485804-d14be980-4371-11ea-9118-ec28a307cf9c.png)

### how it's working now

In each card, there is an "overlay"  link, that through css positioning, covers the whole parent div (like a glass slide in front of the card) and makes the whole DIV clickable. Some of the elements on the card have a `z-index` that bumps them out over the surface of the card (for hover effects).

```
<div class="card card-article card-linked">
 ... card HTML goes here ...

<a class="overlay" href="/2020/01/30/adopting-uswds-one-step-at-time/" title="TITLE"><span></span></a>
</div>
```

Seems that the problem is that the link is technically empty. And that my method of doing this has evolved some https://css-tricks.com/snippets/jquery/make-entire-div-clickable/

### what I need to fix

Either change the cards to be clickable, as they are outlined here: https://css-tricks.com/snippets/jquery/make-entire-div-clickable/

or

get rid of this functionality all together.

**Comment by jeremyzilar** [comment]
Still working on this

**Comment by saracope** [comment]
USWDS released a card component in v2.7. Look at replacing with that.
